src/main/resources/logback-spring.xml

```diff
@@ -8,6 +8,7 @@
         <logger name="jsonLogger" additivity="false" level="DEBUG">
             <appender-ref ref="consoleAppender"/>
         </logger>
+        <logger name="no.nav.finnkandidatapi" level="TRACE"/>
```

Comment:
Trenger vi trace-level nå? :) Eller var det kun for testing av PRen?

Reply:
Ah. Nei, trenger det ikke, men jeg er vant til å ha den på. Fordi jeg tenker at debug- og trace-logging vanligvis skal slettes etter at man er ferdig med den utforsking/feilsøking man holdt på med, typisk rett før merge til master. Slette for å unngå skape støy i koden som skjuler logikken (signal-to-noise ratio). Med denne praksisen slipper jeg å skru trace av og på, og slipper at når jeg skrur trace på så får jeg masse søppellogging som andre har lagt inn før meg.

Gitt at vi er eneinge om at debug og trace-logging skal slettes etter bruk er det bra å ha den enablet, for da blir det synlig at noen ikke har ryddet etter seg.